app/index.sass

```diff
@@ -2,10 +2,11 @@
 $icon-font-path: '../fonts/'
 
 // bower:scss
+// endbower
 @import 'bower_components/bootswatch-scss/flatly/_variables.scss'
 @import 'bower_components/bootstrap-sass-official/assets/stylesheets/_bootstrap.scss'
```

Comment:
since we are using boots watch now we probably should remove all references to bootstrap